pkg/client/options.go

```diff
@@ -388,6 +389,25 @@ func (m MatchingLabels) ApplyToDeleteAllOf(opts *DeleteAllOfOptions) {
 	m.ApplyToList(&opts.ListOptions)
 }
 
+// ContainsLabels filters the list/delete operation checking if the set of labels exists
+// without checking their values.
+type ContainsLabels []string
```

Comment:
@vincepri What about `HasLabels`?  `Contains` to me sounds a bit like it was part of the spec or something

Reply:
sure that works too

Reply:
Fixed!